Portfolio Mobile Version Completion: Add Projects and Skills Section

rmiyoyo opened the pull request:
# Work Done
- Added the project section that has four projects.
- Added About Me page that has languages, frameworks mastered.
- Included customized README template
- Set up config files for Linters and node modules.

rmiyoyo committed:
Add PROJECT page and SKILLS page to Portfolio website

rmiyoyo committed:
Add background images and social media icons

rmiyoyo committed:
Add MIT license and linters config files

ouasamine commented on index.html:
```diff
@@ -0,0 +1,141 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+    <div class="container">
```
- [ ] Your project is currently missing the header and toolbar sections from previous projects, please make sure to implement them, the purpose of the projects is to help build your portfolio gradually, therefore, you need to keep the work from your previous projects.

![2023-03-24_211139](https://user-images.githubusercontent.com/104319462/227644040-eac432bf-d827-459e-9b9b-c8f509c4b5d1.png)

ouasamine commented on style.css:
```diff
@@ -0,0 +1,336 @@
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
```
- [ ] According to the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), you need to use `display: grid` instead of `flex`.

  > In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

ouasamine commented on style.css:
```diff
@@ -0,0 +1,336 @@
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```
- [ ] You need to use `display: flex` to lay out the content of the projects' cards according to the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).

  > In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

ouasamine commented on index.html:
```diff
@@ -0,0 +1,141 @@
+                      <button>HTML</button>
+                      <button>CSS</button>
+                      <button>JAVASCRIPT</button>
+                    </div>
```
- [ ] Here what you need to use instead of a `<div>` is a `<ul>` tag with `<li>` tags instead of `<button>` tags, according to the project's requirements 🔽 

  > Please use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.

ouasamine commented on style.css:
```diff
@@ -0,0 +1,336 @@
+.project-item {
+  border: 1px solid #dfe1e6;
+  width: 327px;
+  height: 596px;
```
- [ ] Please try to use responsive values for `height` and `width` like `%` instead of  `px`, which will improve the responsiveness of your project.

ouasamine commented on index.html:
```diff
@@ -0,0 +1,141 @@
+                      <button>JAVASCRIPT</button>
+                    </div>
+                    <a href="#" class="project-button">See Project</a>
+                  </div>
```
- [ ] Nice work on the project cards, they look good except for the second card where there isn't enough space under the button 🔽 

![2023-03-24_214106](https://user-images.githubusercontent.com/104319462/227646766-522a6d81-502a-4f44-b2d2-e0401376a894.png)

ouasamine commented on index.html:
```diff
@@ -0,0 +1,141 @@
+          
+          <section id="about">
+            <div class="container">
+                <h2>About Myself</h2>
```
- [ ] Here your about section looks a bit different than the Figma design 🔽 , please make sure to change background of your about section to match the Figma design and implement the rounded edge (top-right)

## Your Design
![2023-03-24_214351](https://user-images.githubusercontent.com/104319462/227647781-cdd2c68b-ac8a-4470-965f-39ab06a0f122.png)

## Figma Design
![2023-03-24_214418](https://user-images.githubusercontent.com/104319462/227647793-050990b5-0aab-40cb-b138-89d3ca81d5cb.png)

ouasamine commented on index.html:
```diff
@@ -0,0 +1,141 @@
+                
+                <div class="connect">
+                    <p class="social-media">LET'S CONNECT</p>
+                    <div class="social-icons">
```
- [ ] Please wrap this sub-header "Let's Connect" in a `<hx>` tag instead of `<p>` because it's more semantically proper.

ouasamine commented on index.html:
```diff
@@ -0,0 +1,141 @@
+                            <li class="main-skill"><img src="./images/languages/Skills.png" alt=""></li>
+                        </ul>
+                    </div>
+                </div>
```
- [ ] Nice work here on the skills section but it needs to be further improved, each category eg "Languages", "Frameworks" etc should be a `<li>` tag wrapped inside a `<ul>`, then "languages" should have another `<ul>` tag inside containing all the languages.
- [ ] There is also work still to  be done to match the Figma design 🔽 

## Your Design
![2023-03-24_215847](https://user-images.githubusercontent.com/104319462/227651802-843b2dc7-00a3-4248-89a0-2f7b5b76ad73.png)

## Figma Design
![2023-03-24_215737](https://user-images.githubusercontent.com/104319462/227651809-acd38303-d5a6-456c-a795-c72dff248f4c.png)

ouasamine commented on README.md:
```diff
@@ -0,0 +1,193 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
- [ ] This License link is currently broken, please make sure to fix it.

ouasamine reviewed (changes requested):
Hi @rmiyoyo 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)

## Highlights
- Nice work on the design generally ✔️ 
- Good PR title and description 👏🏽 👏🏽 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rmiyoyo committed:
Add Header and Toolbar sections

rmiyoyo committed:
Rewrite style.css to include both Flex and Grid

rmiyoyo committed:
Update images folder to include JS, HTML, and CSS icons

rmiyoyo commented:
# Work Done After First Review
- Added the Header and Toolbar sections from the first project.
- Included Flex Grid and Flexbox in the sections.
- Changed parameters from pixels to percentage.
- Fixed design issues especially spacing and missing lines.
- Changed "Lets Connect" from ```p``` to ```h2```.
- Fixed broken LICENSE link.
- Aligned my design with the Figma design.

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,204 @@
+            </div>
+            <button class="resume-link">Get my resume</button>
+        </div>
+        <div class="skills-block">
```
- [ ] As suggested by the previous reviewer, your list of technology section is still missing the arrows on the right. Kindly add those arrows so that your design matches the figma designs. Thos arrows are basically there to toggle the technology list 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227702280-a30c87dc-33c5-4b10-b34b-956a784a3d06.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227702315-0b121d67-b9ab-49a5-b6e5-9e120d621cf6.png)

victorokoroji reviewed (changes requested):
Hi @rmiyoyo, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rmiyoyo committed:
Add arrows to the dropdown menus

rmiyoyo commented:
# Work Done After Second Review
- Added the arrows to the technology section.
![image](https://user-images.githubusercontent.com/120647236/227703469-68c56190-fe75-4570-b137-76986266773d.png)

Kamba56 commented on index.html:
```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Mobile Version Completion</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div id="main-div">
+        <header>
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+              <svg viewBox="0 0 100 80" width="40" height="40" class="svg-menu">
+                <rect y="38" width="100" height="8" rx="8" fill="#6070FF"></rect>
+                <rect y="58" width="100" height="8" rx="8" fill="#6070FF"></rect>
+                <rect y="78" width="100" height="8" rx="8" fill="#6070FF"></rect>
+              </svg>
+            </nav>
+        </header>
+        <section id="introduction-paragraphs">
+            <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
+            <p class="description">I'm a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <label class="connect-message">LET'S CONNECT</label>
+
+            <div id="link-media">
+                <ul class="social-icons">
```
- [Optional] Kindly consider using an indentation of two spaces in all your documents. THis is a convention and also best practice.

Kamba56 commented on index.html:
```diff
@@ -0,0 +1,204 @@
+                    <li><label class="tool-press">CSS</label></li>
+                    <li><label class="tool-press">JAVASCRIPT</label></li>
+                </ul>
+                <button class="see-project">See Project</button>
```
- [Optional] Consider adding a hover and pressed state to all your buttons as it is on the figma template. You can use the `:hover` and `:active` pseudoclasses for this.
![image](https://user-images.githubusercontent.com/94551484/227545089-c6ed521a-18ff-4bee-9e00-8f06897df79c.png)

rmiyoyo replied:
@Kamba56 Thank you very much for you input.

Kamba56 reviewed (approved):
Hi @rmiyoyo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏
- [x] Great job using CSS grid and flexbox

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
